src/models/Collectible.js

```diff
@@ -37,7 +37,8 @@ export type Collectible = {
 };
 
 export type CollectibleTransaction = {
-  asset: string,
+  assetSymbol: string,
```

Comment:
You will be amazed to hear that deep down the answer is yes :D however in this case we were using `asset` from any history item in both Archanova and Etherspot history components to map into title of event, but now since it has changed to `assetSymbol` it had to be changed here as well. Completely removing it is kinda yet one more refactor on whole history in the app. TBH `asset` never made sense here (and on `Transaction`) either and it confused a lot with `Asset` type.